apps/framework-cli/src/infrastructure/redis/redis_client.rs

```diff
@@ -386,6 +388,37 @@ impl RedisClient {
         Ok(())
     }
 
+    async fn set<K: ToRedisArgs + Send + Sync, V: ToRedisArgs + Send + Sync>(
+        &self,
+        key: K,
+        value: V,
+    ) -> Result<()> {
+        self.connection
+            .lock()
+            .await
+            .set::<K, V, ()>(key, value)
+            .await
+            .context("Failed to set value in Redis")?;
+        Ok(())
+    }
+
+    pub async fn set_infrastructure_map(
```

Comment:
Does set_infrastructure_map need to be in the RedisClient.  It would be the only application-specific member in the module.  Given that it's just setting a key, a more generic set key function would be better.